chart/templates/services/admin/ingress.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if and .Values.global.huggingface.ingress.enabled .Values.ingress.enabled .Values.admin.ingress.enabled -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  {{- $annotations := fromYaml (include "datasetsServer.instance.ingress.annotations" (dict "instance" .Values.admin "context" $ )) }}
+  annotations: {{ toYaml $annotations | nindent 4 }}
+  labels: {{ include "labels.admin" . | nindent 4 }}
+  name: "{{ include "name" . }}-admin"
+  namespace: {{ .Release.Namespace }}
+spec:
+  rules:
+    - host: {{ include "datasetsServer.ingress.hostname" . }}
+      http:
+        paths:
+          - path: /admin
+            backend:
+              service:
+                name: "{{ include "name" . }}-admin"
+                port:
+                  name: http
+
+            pathType: Prefix
+          {{ if hasKey $annotations "alb.ingress.kubernetes.io/actions.openapi-redirect" -}}
```

Comment:
I think it's not very important, but to avoid problems later (eg if we protect /admin in some way), can we move the openapi redirect to api/ingress.yaml instead?

Reply:
done